docs/BATS.md

```diff
@@ -8,7 +8,7 @@ The otomi/tools:1.4.10 or newer includes the [bats framework](https://github.com
 
 Example of calling the binary:
 
-`docker run --rm otomi/core:latest bats bin/tests`
+`otomi bats`
```

Comment:
Updated documentation. This should make it much simpler.

Reply:
`otomi x bats` ?

Reply:
https://github.com/redkubes/otomi-core/pull/330/files#r581026346

By adding `otomi bats` as shown in the above comment (that you liked) Sebastiaan is able to specify what & how it's tested and outputted. If we go `otomi x bats` then we miss the `-T` and `bin/tests` parameters, and only the `$@` is passed to bats

Reply:
Yes @Dunky13 that's right. I want developers to have immediate feedback at the tip of the fingers.

I also see the possibility for a --dev flag (that's easy to add now) as a compromise (e.g. in `~/.zshrc`: `alias otomi="otomi --dev"`

Reply:
What is wrong with passing extra flags to `otomi x bats`? Why would this:

> then we miss the -T and bin/tests parameters, and only the $@ is passed to bats

be the case?

Reply:
```
    bats)
      drun bats -T "$@" bin/tests
      ;;
```

I could be wrong, but running the load in a container would give us a more consistent environment?

Reply:
Yes, actually, I don't think that we would miss functionality. You're right about that.

It's just a difference in naming the command. I think it's more convenient to have the shortcut across all dev environments. I think `otomi bats` is easier to remember than a second-class citizen like `otomi x bats -T bin/tests`

Reply:
Maybe we should expose dev commands only when one has that role? Or put them not in the docs if that role is not set? Should we introduce a flag `DEV`?

Then we have a straightforward approach to add dev commands and only show them when `DEV` is set

Reply:
Yes, that sounds like a great idea.

I would suggest to add a task as an enhancement, a "dev" mode, because otherwise we will keep this pr open for too long.